src/components/Navbar.jsx

```diff
@@ -1,35 +1,39 @@
 import React, { Component } from "react";
-
+import { Redirect } from "react-router-dom";
 import { connect } from "react-redux";
 import {
   login,
   setLoginError,
   setLoginSuccess,
   setLoginPending,
 } from "../redux/reducer";
-import { Redirect } from "react-router-dom";
+
 class NavBar extends Component {
   constructor(props) {
     super();
   }
-  render() {
-    const isLoggedIn = this.props.login;
 
+  handleLogout = () => {
+    localStorage.removeItem("isLoginSuccess");
+    this.props.setLoginSuccess(false);
+  };
+
+  render() {
+    const serializedState = localStorage.getItem("isLoginSuccess");
+    console.log(serializedState);
+    const isLoggedIn = serializedState;
```

Comment:
const isLoggedIn = localStorage.getItem("isLoginSuccess"); can be used.
remove log